src/components/cuda/tests/Makefile

```diff
@@ -13,12 +13,6 @@ TESTS_NOCTX = concurrent_profiling_noCuCtx pthreads_noCuCtx \
 			  simpleMultiGPU_noCuCtx
 
 NVCC = $(PAPI_CUDA_ROOT)/bin/nvcc
-NVCC_VERSION := $(shell $(NVCC) --version | grep -oP '(?<=release )\d+\.\d+')
-ifeq ($(shell echo "$(NVCC_VERSION) >= 11.6" | bc), 1)
-    NVCFLAGS := -arch=native
-else
-    NVCFLAGS :=
-endif
 
 PAPI_FLAG = -DPAPI    # Comment this line for tests to run without PAPI profiling
 NVCFLAGS += -g -ccbin='$(CC)' $(PAPI_FLAG)
```

Comment:
Now that you removed the initial setting of `NVCFLAGS`, you should also remove the `+` from line:
 `NVCFLAGS += -g -ccbin='$(CC)' $(PAPI_FLAG)` so that it looks like:
`NVCFLAGS = -g -ccbin='$(CC)' $(PAPI_FLAG)`

Reply:
Thank you for pointing that out. I have pushed this change up to the PR just now.